File: example/java/jimmer-cloud/registry-center/src/main/java/org/babyfish/jimmer/example/cloud/registry/RegistryCenterApp.java
package org.babyfish.jimmer.example.cloud.registry;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;

@SpringBootApplication
@EnableEurekaServer
public class RegistryCenterApp {

    public static void main(String[] args) {
        SpringApplication.run(RegistryCenterApp.class, args);
    }
}

File: example/kotlin/jimmer-cloud-kt/registry-center/src/main/kotlin/org/babyfish/jimmer/example/cloud/kt/RegistryCenterApp.kt
package org.babyfish.jimmer.example.cloud.kt

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer

@SpringBootApplication
@EnableEurekaServer
class RegistryCenterApp

fun main(args: Array<String>) {
    runApplication<RegistryCenterApp>(*args)
}